Error [nuxt] [request error] [unhandled] [500] Cannot find module './compose/composer.js'

![Image](https://github.com/user-attachments/assets/a73e6c49-7672-4e2b-9f1f-65d140f2a08c)

 I encountered an error while using the latest version of Nuxt (^3.15.0).

![Image](https://github.com/user-attachments/assets/60bd0051-7b0e-467e-8702-8ee0718920c2)

From this morning I have the same error

```
 WARN  [Vue Router warn]: uncaught error during route navigation:


 ERROR  Cannot find module './compose/composer.js'
Require stack:
- /home/.../github.com/.../.../yaml

    Require stack:
    - yaml
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at yaml:3:16
    at ViteNodeRunner.runModule (node_modules/.pnpm/vite-node@3.0.4_@types+node@22.10.10_jiti@2.4.2_lightningcss@1.29.1_terser@5.37.0_tsx@4.19.2_yaml@2.7.0/node_modules/vite-node/dist/client.mjs:399:11)
    at ViteNodeRunner.directRequest (node_modules/.pnpm/vite-node@3.0.4_@types+node@22.10.10_jiti@2.4.2_lightningcss@1.29.1_terser@5.37.0_tsx@4.19.2_yaml@2.7.0/node_modules/vite-node/dist/client.mjs:381:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ViteNodeRunner.cachedRequest (node_modules/.pnpm/vite-node@3.0.4_@types+node@22.10.10_jiti@2.4.2_lightningcss@1.29.1_terser@5.37.0_tsx@4.19.2_yaml@2.7.0/node_modules/vite-node/dist/client.mjs:206:14)
    at async ViteNodeRunner.dependencyRequest (node_modules/.pnpm/vite-node@3.0.4_@types+node@22.10.10_jiti@2.4.2_lightningcss@1.29.1_terser@5.37.0_tsx@4.19.2_yaml@2.7.0/node_modules/vite-node/dist/client.mjs:259:12)
```

This is my package.json

```
"dependencies": {
    "@nuxt/content": "3.0.0",
    "@nuxt/eslint": "0.7.4",
    "@nuxt/image": "1.9.0",
    "@nuxt/scripts": "0.9.5",
    "@nuxt/ui-pro": "https://pkg.pr.new/@nuxt/ui-pro@2dbbff0",
    "@nuxtjs/i18n": "9.0.0-rc.2",
    "@nuxtjs/turnstile": "0.9.11",
    "eslint": "^8.57.0 || ^9.0.0",
    "nuxt": "^3.15.3",
    "nuxt-og-image": "4.0.2",
    "nuxt-site-config": "^3.0.6",
  },
```

.npmrc
```
shamefully-hoist=true
strict-peer-dependencies=false
```

Hello,

can also confirm I have the same error this morning since upgrading from `3.15.2` to `3.15.3`:

```
 ERROR  [nuxt] [request error] [unhandled] [500] Cannot find module './compose/composer.js'
Require stack:
- /Users/[..]/my-app/yaml
```

.npmrc

```
shamefully-hoist=true
strict-peer-dependencies=false
```

--- a/src/module.ts
+++ b/src/module.ts
@@ -68,6 +68,9 @@ export default defineNuxtModule<ModuleOptions>({
       headings: options.headings!
     })
 
+    nuxt.options.build.transpile ||= []
+    nuxt.options.build.transpile.push('yaml')
+
     if (options.highlight) {
       // Enable unwasm for shiki
       addWasmSupport(nuxt)

